jina/orchestrate/flow/base.py

```diff
@@ -147,11 +147,15 @@ def __init__(
         deployments_addresses: Optional[str] = '{}',
         deployments_disable_reduce: Optional[str] = '[]',
         description: Optional[str] = None,
+        disable_auto_volume: Optional[bool] = False,
+        docker_kwargs: Optional[dict] = None,
+        entrypoint: Optional[str] = None,
         env: Optional[dict] = None,
         exit_on_exceptions: Optional[List[str]] = [],
         expose_endpoints: Optional[str] = None,
         expose_graphql_endpoint: Optional[bool] = False,
         floating: Optional[bool] = False,
+        gpus: Optional[str] = None,
```

Comment:
I am not sure ti understand why we have gpu here

Reply:
gateway parser now has container capability and this will be fixed once we separate flow and gateway API here: https://github.com/jina-ai/jina/issues/5215